src/action-creators/cursorDown.js

```diff
@@ -8,18 +8,17 @@ import {
   isDivider,
   restoreSelection,
   selectNextEditable,
+  getThoughtsRanked,
 } from '../util.js'
 
 export const cursorDown = ({ target }) => dispatch => {
-
   const { cursor } = store.getState()
-
   if (cursor) {
     // select next editable
     const nextThought = getThoughtAfter(cursor)
-
+    const descendants = getThoughtsRanked(cursor)
```

Comment:
`getThoughts` is more efficient than `getThoughtsRanked` since it does not sort.

Also, you changed the function call but didn't change the variable name to match. Please try to pay closer attention to details like this.